### DISCUSSION TITLE
resource with no memory

### DISCUSSION THREAD

**Discussion by yangvipguang** [prometheus-operator/kube-prometheus#625] [General] [answered]
I have configured the resource related configuration in the jsonnet file, but after compiling, I only see the cpu resource, not the memory resource。
![image](https://user-images.githubusercontent.com/11416887/89007766-f844a180-d33b-11ea-81ef-5f3e2081ca3b.png)

![image](https://user-images.githubusercontent.com/11416887/89007784-ff6baf80-d33b-11ea-8a10-147b4c095cb0.png)

**Comment by brancz** [comment, marked as answer]
you need to put both of them in the same object as in `{ memory: '1024Mi', cpu: '500m" }`
